index.html

```diff
@@ -177,10 +177,13 @@ <h3 class="sixth-sheet-developer-text">Skills</h3>
     <h2 class="seventh-sheet__contact-me">Contact me</h2>
     <p class="seventh-sheet__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
     <form action="https://formspree.io/f/" class="form" method="post">
-      <input class="form__field-area form__field-text" name="name" type="text" id="name" maxlength="30" placeholder="Your name" required>
-      <input class="form__field-area form__field-text" name="email" type="email" id="e-mail" placeholder="user_name@mail.com" required>
-      <textarea class="form__field-area-message" name="area" id="area" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
-      <button type="submit" class="seventh-sheet__button seventh-sheet__button-text">Get It Touch</button>
+      <img class="form__image" src="./images/contact-form-background.svg" alt="background-form">
```

Comment:
- [ ] This image is overflowing the form fields so the form fields and button are not clickable. Please make sure your form is working and all form fields are accessible.
   - I would suggest using this image in the background of this contact section using the CSS `background-image `property.